src/qiskit_sphinx_theme/__init__.py

```diff
@@ -71,13 +71,25 @@ def activate_themes(app: sphinx.application.Sphinx, config: sphinx.config.Config
     if config.html_theme == "qiskit":
         # We set a low priority so that our Qiskit CSS file overrides Furo.
         app.add_css_file("styles/furo.css", 100)
-        app.add_js_file("scripts/furo.js")
```

Comment:
Turns out, this line was never necessary. We already ran it as part of Furo's `__init__.py`. We only need to add back `furo.css` because they load it via `theme.conf`, not `__init__.py`.